system-tests/azure-tests/src/test/java/org/eclipse/dataspaceconnector/system/tests/local/BlobTransferIntegrationTest.java

```diff
@@ -0,0 +1,239 @@
+import static org.eclipse.dataspaceconnector.system.tests.local.TransferLocalSimulation.CONSUMER_IDS_API;
+import static org.eclipse.dataspaceconnector.system.tests.local.TransferLocalSimulation.CONSUMER_IDS_API_PORT;
+import static org.eclipse.dataspaceconnector.system.tests.local.TransferLocalSimulation.CONSUMER_MANAGEMENT_PATH;
+import static org.eclipse.dataspaceconnector.system.tests.local.TransferLocalSimulation.CONSUMER_MANAGEMENT_PORT;
+import static org.eclipse.dataspaceconnector.system.tests.local.TransferLocalSimulation.PROVIDER_CONNECTOR_MANAGEMENT_URL;
+import static org.eclipse.dataspaceconnector.system.tests.local.TransferLocalSimulation.PROVIDER_CONNECTOR_PATH;
+import static org.eclipse.dataspaceconnector.system.tests.local.TransferLocalSimulation.PROVIDER_CONNECTOR_PORT;
+import static org.eclipse.dataspaceconnector.system.tests.local.TransferLocalSimulation.PROVIDER_IDS_API;
+import static org.eclipse.dataspaceconnector.system.tests.local.TransferLocalSimulation.PROVIDER_IDS_API_PORT;
+import static org.eclipse.dataspaceconnector.system.tests.local.TransferLocalSimulation.PROVIDER_MANAGEMENT_PATH;
+import static org.eclipse.dataspaceconnector.system.tests.local.TransferLocalSimulation.PROVIDER_MANAGEMENT_PORT;
+import static org.eclipse.dataspaceconnector.system.tests.utils.GatlingUtils.runGatling;
+import static org.eclipse.dataspaceconnector.system.tests.utils.TransferSimulationUtils.PROVIDER_ASSET_FILE;
+import static org.eclipse.dataspaceconnector.system.tests.utils.TransferSimulationUtils.PROVIDER_ASSET_ID;
+import static org.eclipse.dataspaceconnector.system.tests.utils.TransferSimulationUtils.TRANSFER_PROCESSES_PATH;
+
+@AzureStorageIntegrationTest
+public class BlobTransferIntegrationTest extends AbstractAzureBlobTest {
+    private static final Vault CONSUMER_VAULT = new MockVault();
+    private static final Vault PROVIDER_VAULT = new MockVault();
+    private static final String ASSETS_PATH = "/assets";
+    private static final String POLICIES_PATH = "/policies";
+    private static final String CONTRACT_DEFINITIONS_PATH = "/contractdefinitions";
+    private static final String PROVIDER_CONTAINER_NAME = UUID.randomUUID().toString();
+
+    @RegisterExtension
+    protected static EdcRuntimeExtension consumer = new EdcRuntimeExtension(
+            ":system-tests:runtimes:azure-storage-transfer-consumer",
+            "consumer",
+            Map.of(
+                    "edc.blobstore.endpoint.template", "http://127.0.0.1:10000/%s",
+                    "web.http.port", String.valueOf(CONSUMER_CONNECTOR_PORT),
+                    "web.http.path", CONSUMER_CONNECTOR_PATH,
+                    "web.http.data.port", String.valueOf(CONSUMER_MANAGEMENT_PORT),
+                    "web.http.data.path", CONSUMER_MANAGEMENT_PATH,
+                    "web.http.ids.port", String.valueOf(CONSUMER_IDS_API_PORT),
+                    "web.http.ids.path", "/api/v1/ids",
```

Comment:
nit: `/api/v1/ids` can also be added constant and used like others e.g. CONSUMER_MANAGEMENT_PATH